bitnami/mongodb/templates/secrets.yaml

```diff
@@ -9,22 +9,22 @@ metadata:
 type: Opaque
 data:
   {{- if .Values.auth.rootPassword }}
-  mongodb-root-password:  {{ .Values.auth.rootPassword | b64enc | quote }}
+  mongodb-root-password:  {{ .Values.auth.rootPassword | toString | b64enc | quote }}
   {{- else }}
-  mongodb-root-password: {{ randAlphaNum 10 | b64enc | quote }}
+  mongodb-root-password: {{ randAlphaNum 10 | toString | b64enc | quote }}
```

Comment:
For the ones that are auto-generated this is not needed.

Reply:
Have updated this now.